daghelpers_test.go

```diff
@@ -0,0 +1,144 @@
+package format
+
+import (
+	"context"
+	"github.com/ipfs/go-cid"
+	mh "github.com/multiformats/go-multihash"
+	"github.com/stretchr/testify/assert"
```

Comment:
We try not to add *too* many external deps unless we need them. I'd prefer it if we could stick with go's testing tools.